.github/workflows/CI.yml

```
@@ -0,0 +1,119 @@
+name: Ubuntu env 
```

```suggestion
name: Continuous Integration for Machine Learning Applications
```
something more descriptive should be better.